packages/lumx-react/src/components/date-picker/DatePicker.tsx

```diff
@@ -57,7 +57,7 @@ const DEFAULT_PROPS: Partial<DatePickerProps> = {
  * @return The component.
  */
 const DatePicker = (props: DatePickerProps) => {
-    const today = props.value || moment();
+    const today = moment(props.value);
```

Comment:
We have to be sure that `today` equals `moment()` if the value is `undefined` or if the string value is not parseable.